### ISSUE TITLE
[Suggestion] mechanism link to mechanism 

### ISSUE THREAD

**Issue by Tackietacktack** [tsunamayo/Starship-EVO#5998]
Hi, small idea.
If we want, that two piston moves symmetrical, we need a many time to set the piston speed fine.

The idea is that we can link piston to other piston. All linked piston could moves symetrical(same speed and length).

Same by hinge and rotator.

What do you think?

**Comment by ProPeach** [comment]
This would be great, as it's often impossible to get mechanisms to travel at exactly the same speed due to mass differences affecting their speed.
This causes problems when for example you want two pistons to extend a landing gear foot out together. The one that does not have the foot built on it will be faster because it is not pushing any mass, and you're unable to match the precise speed of the piston with the foot on accurately enough with just the %slider. So when you activate them both at the same time, there is a collision and the foot fails to extend

Ideally I would like to see a system where you input the precise speed into a text box in meters/s or degrees/s for example rather than using this linking system, but I would welcome it regardless

**Comment by Majikmonster** [comment]
> Ideally I would like to see a system where you input the precise speed into a text box in meters/s or degrees/s for example rather than using this linking system, but I would welcome it regardless

Agreed. The slider we have is very imprecise and even the smallest weight differences cause timing issues. Manual integer input is ideal. Also, it would help if there were a minimum weight limit where mechanism speed is unaffected until N% of its max load is reached, rather than every single kg on any scale of mechanism slowing it down.

(ie. a 1/8th light tile shouldn't cause a 32m hinge to slow down at all)
